frontend/src/components/elements/Badge/Badge.styled.js

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components';
+import { colorGrey, colorGreyLight, colorBlack } from '../../../styles/colors';
+
+const getBadgeColor = props => {
+  let baseColor = colorGreyLight;
+  return props.isHighlighted ? colorGrey : baseColor;
+};
+
+const AutoCompleteBadgeStyled = styled.div`
+  max-width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  margin: 0 2px 2px 0;
+  border-radius: 1px;
+  cursor: pointer;
+
+  p:first-of-type {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    padding: 0 1.5rem;
+
+    color: ${colorBlack};
+    font-size: 1.5rem;
+    font-weight: bold;
+  }
+
+  background-color: ${props => getBadgeColor(props)};
+`;
+
+const BadgeStyled = styled.div`
+  max-width: 100%;
+  height: 1.6rem;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  justify-content: space-between;
+
+  margin: 0 2px 2px 0;
+  border-radius: 1px;
+  padding: 10px 0px;
+
```

Comment:
```suggestion
   box-sizing: content-box;
```
adding this here should fix the text popping out.